src/scripts/active-inactive-orgunit/index.js

```diff
@@ -74,22 +67,25 @@ OrganisationUnitActivator.prototype.prepareOptions = function (endpoint) {
     options.url += endpoint;
     return options;
 }
+
 /**
  * Prepares the data Elements and loads the organisationUnits for each dataelement in a given data element group
+@param dataElement The active dataElement
```

Comment:
This param doesn't correspond to any actual param in the function. What is more, there are two params and none of them is in the comment